engine/src/python/config/python_utils.cpp

```diff
@@ -65,18 +64,51 @@ PyObject* GetClassFromPython(
         // TODO: throw exception
         return nullptr;
     }
-    
+
+    PyPreConfig py_pre_config;
+    PyPreConfig_InitPythonConfig(&py_pre_config);
+
+    PyStatus status;
+
+    status = Py_PreInitialize(&py_pre_config);
+    if (PyStatus_Exception(status)) {
+        Py_ExitStatusException(status);
+    }
+
+    const std::string python_path = GetPythonPath();
+    const std::wstring python_path_w(python_path.begin(), python_path.end());
+    const std::wstring path_string_w = std::wstring(path_string.begin(), path_string.end());
     const std::string yaml_path = QUOTE(Python_SITELIB);
-    const std::string python_path_string = GetPythonPath() 
-                                           + ":" + path_string
-                                           + ":" + yaml_path + ":" + build_path;
-    const std::wstring python_path_wstring = std::wstring(python_path_string.begin(), 
-                                                          python_path_string.end());
-    const wchar_t* python_path = python_path_wstring.c_str();
-    Py_SetPath(python_path);
-    Py_Initialize();
+    const std::wstring yaml_path_wstring = std::wstring(yaml_path.begin(), yaml_path.end());
```

Comment:
I thought we agreed to drop the yaml. AFAIK it's the python sitelib, so we should rename it if we're keeping it.

Reply:
Oh, you're right. Let me fix that.

Reply:
Fixed.